### DISCUSSION TITLE
Add VISA dataset to anomalib

### DISCUSSION THREAD

**Discussion by ORippler** [open-edge-platform/anomalib#752] [Feature Requests] [answered]
It would be awesome to integrate the [VISA](http://github.com/amazon-research/spot-diff) dataset into anomalib. It's much bigger and supposedly much more difficult than MVTec AD, where we can see saturation setting in.

**Comment by djdameln** [comment]
I personally was not aware of this dataset, so thanks for the suggestion!

We've been looking for a more challenging benchmark dataset, as the saturation effect you mentioned is making MVTec less suited for rating performance. This definitely looks like it could be a good addition to MVTec to use in benchmarking experiments.

**Comment by samet-akcay** [comment, marked as answer]
@ORippler, thanks for the suggestion!  From the first glance, I can say this is a lot more realistic use case.

Yes, we should definitely add this ASAP!

**Comment by samet-akcay** [comment]
https://anomalib.readthedocs.io/en/latest/markdown/guides/reference/data/image/visa.html
